ownCloud/Key Commands/KeyCommands.swift

```diff
@@ -274,6 +275,9 @@ extension UITableViewController {
 	@objc func selectNext(sender: UIKeyCommand) {
 		if let selectedIndexPath = self.tableView?.indexPathForSelectedRow, selectedIndexPath.row < (self.tableView?.numberOfRows(inSection: selectedIndexPath.section) ?? 0 ) - 1 {
 			self.tableView.selectRow(at: NSIndexPath(row: selectedIndexPath.row + 1, section: selectedIndexPath.section) as IndexPath, animated: true, scrollPosition: .middle)
+		} else if let selectedIndexPath = self.tableView?.indexPathForSelectedRow, (selectedIndexPath.section + 1) < self.tableView?.numberOfSections ?? 0 {
```

Comment:
@hosy for readability I would suggest to add some guard let statements in the beginning for all the optional stuff

Reply:
Fixed!